servers/rendering/renderer_rd/shaders/ffx_a.h

```diff
@@ -0,0 +1,4987 @@
+//==============================================================================================================================
+//
+//                                               [A] SHADER PORTABILITY 1.20210629
+//
+//==============================================================================================================================
+// FidelityFX Super Resolution Sample
+//
+// Copyright (c) 2021 Advanced Micro Devices, Inc. All rights reserved.
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
```

Comment:
Note that it's not a correct beginning of the (non-third party) header file which makes static check failed. You can take a proper header from any other header file across the engine.

Reply:
This needs to be using the standard Godot Engine license header or it can't be merged.

Reply:
I also think it should be moved to third party, but I'm not sure the exact workflow for drivers. (things that aren't modules but need to be in core.)

Reply:
"ffx_a.h" and "ffx_fsr1.h" have been moved to "thirdparty/amd-fsr"